registration/include/pcl/registration/sample_consensus_prerejective.h

```diff
@@ -285,8 +301,21 @@ namespace pcl
         * \param inliers indices of source point cloud inliers
         * \param fitness_score output fitness score as RMSE 
         */
+      PCL_DEPRECATED(1, 15, "Please use `getFitness(final_transformation_, inliers)` instead")
       void 
-      getFitness (std::vector<int>& inliers, float& fitness_score);
+      getFitness (std::vector<int>& inliers, float& fitness_score) const;
+
+      /** \brief Obtain the fitness of a transformation
+        * The following metrics are calculated, based on
+        * \b final_transformation and \b corr_dist_threshold_:
```

Comment:
Should this `final_transformation` be `transformation`?

Reply:
Thanks. Fixed.